docs/mr_proc/workflow/proc_pipe/mriqc.md

```diff
@@ -0,0 +1,50 @@
+### MRIQC image processing pipeline
+
+---
+
+MRIQC processes the participants and produces image quality metrics from T1w, T2w and BOLD data.
+
+---
+
+
+### [MRIQC](https://mriqc.readthedocs.io/en/latest/)
+- Ensure you are running the appropriate MRIQC image, as provided [here](https://github.com/neurodatascience/mr_proc/blob/main/workflow/proc_pipe/mriqc/Dockerfile)
+- Use [run_mriqc.py](https://github.com/neurodatascience/mr_proc/tree/main/workflow/proc_pipe/mriqc) to run MRIQC pipeline directly or wrap the script in an SGE/Slurm script to run on cluster
+
+python run_mriqc.py --global_config CONFIG.JSON --subject_id 001 --output_dir OUTPUT_DIR_PATH
+	- Mandatory: Pass in the absolute path to the configuration containing the MRIQC container and data directory to `global_config`
+	- Mandatory: Pass in the subject id to `subject_id`
+	- Mandatory: Pass in the absolute path to the output directory to `output_dir`
+	
+!!! note
+	An example config is located [here](https://github.com/neurodatascience/mr_proc/blob/main/sample_global_configs.json)
+
+> Sample cmd:
+```bash
+python run_mriqc.py \
+ 	--global_config GLOBAL_CONFIG \
+ 	--subject_id SUBJECT_ID \
+ 	--output_dir OUTPUT_DIR
```

Comment:
I think we are not requiring `--session_id` for all run scripts. 
Also for consistency, let's name `--subject_id` as `participant_id`

See [run_fmriprep.py](https://github.com/neurodatascience/mr_proc/blob/main/workflow/proc_pipe/fmriprep/run_fmriprep.py#L139-L140) for example